src/App.js

```diff
@@ -15,7 +15,7 @@ function App() {
       <Sidebar />
       <div className="container">
         <Routes>
-            <Route exact path="/" element={<FormElement obj = {config["config"]["apis"]["create"]} title = "Create Partner" description = "This Api helps you to create a partner in the system"/>}> 
+            <Route exact path="/" element={<FormElement obj = {config["config"]["apis"]["getSubscription"]} title = "Get Subscription ID" description = "This Api helps you to get the subscription id."/>}> 
```

Comment:
/** TODO: 
* sub routing
**/

Reply:
error boundary to check config value https://reactjs.org/docs/error-boundaries.html